tests/forte/data/data_store_serialization_test.py

```diff
@@ -36,86 +39,111 @@ def setUp(self) -> None:
         DataStore._type_attributes = {
             "ft.onto.base_ontology.Document": {
                 "attributes": {
-                    "sentiment": 4,
-                    "classifications": 5,
+                    "begin": 2,
```

Comment:
What if begin and end are not included in "attributes"? Will it trigger an error.

Reply:
Yes I think they will, since `begin` and `end` are attributes of `Annotation`, they are required to initialize a data store entry (since they are stored in a sorted list based on the `begin` and `end`) and also in other operations where we fetch the index of the position where these attributes are stored (using `get_datastore_attr_idx`)